lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -98,7 +98,7 @@ public function GetQueueContents($ids, $order = 'createtime,desc', $state = NULL
                 break;
         }
 
-        if(!ConfigHelper::checkConfig('privileges.superuser'))
+        if(!ConfigHelper::checkPrivilege('superuser') && !ConfigHelper::checkPrivilege('helpdesk_extend_operation'))
```

Comment:
To można zastąpić przez użycie pojedynczego wywołania:
https://github.com/lmsgit/lms/blob/master/lib/LMSConfig/ConfigHelper.php#L127-L131
czyli:
```php
if (!ConfigHelper::checkPrivilege('helpdesk_advanced_operations'))
```